### PR TITLE
Add statespace tests

### PR DESCRIPTION
**Pull request by norhh**
I always wondered why i didn't need to update the graph tests after each change.Graph tests were testing for exceptions and they check `assert [] == []` at the end, will remove graph tests files after this PR.

### CODE REVIEW

File: tests/graph_test.py [hunk 1]
```diff
@@ -1,45 +1,22 @@
...
-class GraphTest(BaseTestCase):
-    def test_generate_graph(self):
-        for input_file in TESTDATA_INPUTS.iterdir():
-            output_expected = TESTDATA_OUTPUTS_EXPECTED / (
-                input_file.name + ".graph.html"
-            )
-            output_current = TESTDATA_OUTPUTS_CURRENT / (
-                input_file.name + ".graph.html"
-            )
+def test_generate_graph():
```
Comment (nbanmp) [inline tests/graph_test.py]: Since you removed checking `input_file.graph.html`, do you want to you also remove the files themselves?

Comment (norhh) [reply tests/graph_test.py]: Yeah, I planned on removing the directory after this get's merged. Or I can remove it now

**Comment by JoranHonig** [comment]
With this pr there are no assertions anymore right?
So the only thing that would be captured is an exceptional halt for the generation of the graphs?

**Comment by norhh** [comment]
` So the only thing that would be captured is an exceptional halt for the generation of the graphs?` Yes.

**Comment by JoranHonig** [comment]
@norhh okay, do you think we should maybe add a disclaimer along the lines of "This test only checks whether dumping is successfull, not whether the dumped statespace makes sense"

**Pull request merged**